frontend/src/assets/scss/typography.scss

```diff
@@ -2,47 +2,53 @@ html body {
   margin: 0;
   padding: 0;
   height: 100%;
-  font-family: "Inter", sans-serif;
+  font-family: "Open Sans", sans-serif;
   font-weight: 400;
   -webkit-font-smoothing: antialiased;
 }
 
 @supports (font-variation-settings: normal) {
   html {
-    font-family: "Inter var", sans-serif;
+    font-family: "Open Sans", sans-serif;
   }
 }
 
 // Headings
 
 h1 {
-  @apply text-4xl font-semibold leading-normal;
+  font-family: "Roboto Slab", sans-serif;
+  @apply text-4xl font-semibold leading-normal uppercase;
 }
 
 h2 {
-  @apply text-3xl font-semibold leading-normal;
+  font-family: "Roboto Slab", sans-serif;
+  @apply text-3xl font-semibold leading-normal uppercase;
 }
 
 h3 {
-  @apply text-2xl font-semibold leading-normal;
+  font-family: "Roboto Slab", sans-serif;
+  @apply text-2xl font-semibold leading-normal uppercase;
 }
 
 h4 {
-  @apply text-xl font-semibold leading-normal;
+  font-family: "Roboto Slab", sans-serif;
+  @apply text-xl font-semibold leading-normal uppercase;
 }
 
 h5 {
-  @apply text-lg font-semibold leading-normal;
+  font-family: "Roboto Slab", sans-serif;
+  @apply text-lg font-semibold leading-normal uppercase;
 }
 
 h6 {
+  font-family: "Open Sans", sans-serif;
```

Comment:
Is it intended for the h6 heading style to have a `Open Sans` font, while all other heading styles use `Roboto Slab`?

Reply:
Yeah I've aligned on this with Nuno.
So all headings should have Roboto Slab font. But for the smallest ones it's ok to maintain Open Sans